src/modules/documentExecution/main/DocumentExecution.vue

```diff
@@ -182,6 +184,8 @@ import DashboardController from '../dashboard/DashboardController.vue'
 import Dialog from 'primevue/dialog'
 import descriptor from './DocumentExecutionDescriptor.json'
 import UserFunctionalitiesConstants from '@/UserFunctionalitiesConstants.json'
+import WorkspaceFolderPickerDialog from './dialogs/workspaceFolderPickerDialog/WorkspaceFolderPickerDialog.vue'
+import mockedView from './mockedView.json'
```

Comment:
Can you remove it if unused, please? Thank you.

Reply:
Fixed